guide/ops/https.md

```diff
@@ -0,0 +1,53 @@
+---
+title: HTTPS Configuration
+layout: website-normal
+---
+
+## HTTPS Configuration
+
+### Getting the Certificate
+To enable HTTPS web access, you will need a server certificate in a java keystore. To create a self-signed certificate,
```

Comment:
Perhaps link out [java keystore](https://en.wikipedia.org/wiki/Keystore) and say something like - To create a self-signed certificate (for testing and non-production use)